yarn.lock

```diff
@@ -23,7 +23,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@babel/code-frame@npm:^7.10.3, @babel/code-frame@npm:^7.16.7":
+"@babel/code-frame@npm:^7.10.3, @babel/code-frame@npm:^7.16.7, @babel/code-frame@npm:^7.5.5":
```

Comment:
this looks funky, not sure why this would cause all these changes to the lockfile 🤔

Reply:
not sure what happend, synced with main and copied yarn.lock file and then a yarn install and now the diff is gone. 

And now the diff line totals make more sense (removing lines of code).  I think I can remove more, there are some types left over from the old state handling. 

<img width="171" alt="Screenshot 2022-11-01 at 19 38 28" src="https://user-images.githubusercontent.com/10999/199312076-70956a1e-f7f6-47f5-9a54-7c50bc56e04c.png">